Neos.Cache/Classes/Backend/BackendInterface.php

```diff
@@ -56,7 +56,7 @@ public function getPrefixedIdentifier($entryIdentifier);
      * @throws \Neos\Cache\Exception\InvalidDataException if $data is not a string
```

Comment:
With `$data` being declared as `string` that exception can never be thrown anymore, no? Might be the case for other places as well…